GSA_Adapter/GSAAdapter.cs

```diff
@@ -43,7 +43,7 @@ public partial class GSAAdapter : BHoMAdapter
 
         public GSAAdapter(string filePath = "", GSAConfig gsaConfig = null, bool active = false)
         {
-            AdapterIdName = BH.Engine.GSA.Convert.AdapterIdName;
+            AdapterIdName = Convert.AdapterIdName;
```

Comment:
As per my comment above, I see no reason for this to happen anymore - I'd prefer to see this as a field of the Adapter itself.

Reply:
see comment above

Reply:
Same